actions.csv

```diff
@@ -1,180 +1,187 @@
-101,2019/01/18,https://www.theguardian.com/technology/2019/jan/18/oracle-women-workers-lawsuit-salaries-pay,legal_action,discrimination,"Attorneys seek to represent more than 4,200 women in a lawsuit against Oracle that alleged that female employees were paid on average $13,000 less per year than men doing similar work. An analysis of payroll data found disparities with an ""extraordinarily high degree of statistical significance"", the complaint said. Women made 3.8% less in base salaries on average than men in the same job categories, 13.2% less in bonuses, and 33.1% less in stock value, it alleges.","redwood_shores, california, usa",oracle,4200,sexism,organizejs
-102,2019/02/01,https://www.nottinghampost.com/news/nottingham-news/deliveroo-riders-strike-nottingham-city-2498758,protest,"working_conditions, pay_and_benefits","Deliveroo riders took to Nottingham's Speakers' Corner to strike in a pay dispute. Riders were striking over the fees paid for long and short distance deliveries in the Nottingham area. Riders receive no pension, holiday or sickness pay, and must cover their own equipment costs and expenses and pay national insurance and tax on their earnings.","nottingham, uk",deliveroo,30,"delivery_workers, independent_contract_workers",organizejs
-103,2019/02/18,https://clb.org.hk/content/food-delivery-drivers-strike-protest-post-new-year-pay-cuts,strike,pay_and_benefits,"Food delivery workers in at least five cities across China have staged a series of strikes protesting arbitrary reductions in pay rates introduced after the Lunar New Year Holiday. Workers for both major food delivery platforms, Meituan and Ele.me, stopped taking orders starting in February.","linyi, dongguan, jinan, pinghu, qingdao, china","meituan, ele.me",None,"delivery_workers, independent_contract_workers",organizejs
-104,2019/02/22,"https://slate.com/technology/2019/02/microsoft-workers-protest-hololens-pentagon-contract.html, https://www.buzzfeednews.com/article/carolineodonovan/microsoft-hololens-army",open_letter,ethics,"Microsoft employees circulated a letter among the companies' over 130,000-person staff demanding that executives cancel a $479 million contract with the US Army, IVAS, that will provide weapons technology to the U.S. Military to 'increase lethality' to army soldiers. ""We did not sign up to develop weapons, and we demand a say in how our work is used.","online, usa",microsoft,290,"white_collar_workers, military_contracts",organizejs
-105,2019/02/27,https://www.buzzfeednews.com/article/carolineodonovan/student-groups-protest-salesforce-palantir-ice-campus,protest,ethics,"Ten Stanford students with a group called Students for the Liberation of All People (SLAP) protested a guest lecture by Salesforce executive Richard Socher. The students, who held signs — ""Salesforcing families apart,"" ""Richard are you getting rich from deportation?"" a
```

Comment:
if we're including this (not explicitly by tech workers), we should look at some of the net neutrality campaigns that groups like fight for the future has done

Reply:
Good idea!